independent-publisher-connectors/openFDA Drug/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1104 @@
+                            },
+                            "reference_drug": {
+                              "type": "string",
+                              "description": "Indicates whether the drug product is a reference drug."
+                            },
+                            "brand_name": {
+                              "type": "string",
+                              "description": "Brand or trade name of the drug product."
+                            },
+                            "active_ingredients": {
+                              "type": "array",
+                              "items": {
+                                "type": "object",
+                                "properties": {
+                                  "name": {
+                                    "type": "string",
+                                    "description": "The names of the active, medicinal ingredients in the drug product."
+                                  },
+                                  "strength": {
+                                    "type": "string",
+                                    "description": "The strength of the active, medicinal ingredients in the drug product."
+                                  }
+                                }
+                              },
+                              "description": "Active ingredients"
+                            },
+                            "reference_standard": {
+                              "type": "string",
+                              "description": "Indicates whether the drug product is a reference standard."
+                            },
+                            "dosage_form": {
+                              "type": "string",
+                              "description": "The drug’s dosage form. There is no standard, but values may include terms like tablet or solution for injection."
+                            },
+                            "route": {
+                              "type": "string",
+                              "description": "The route of administation of the drug product."
+                            },
+                            "marketing_status": {
+                              "type": "string",
+                              "description": "Indicates how a drug product is sold in the United States."
+                            }
+                          }
+                        },
+                        "description": "Products"
+                      }
+                    }
+                  },
+                  "description": "Results"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Drugs@FDA",
```

Comment:
@last72 Please provide a valid sentence for summary value.

Documentation - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@sriyen-msft Special char "@" removed